python/ray/air/util/tensor_extensions/pandas.py

```diff
@@ -1361,3 +1384,44 @@ def _is_boolean(self):
 TensorArray._add_arithmetic_ops()
 TensorArray._add_comparison_ops()
 TensorArray._add_logical_ops()
+
+
+@PublicAPI(stability="beta")
```

Comment:
why this is a public API? are we expecting users to use it? Should it be a developer API or without API annotation?

Reply:
Ah good point, this was back from before `TensorArray` could support both non-variable shaped and variable shaped tensors, when the user would need to do their own check to cast their tensor to the right extension type. I'll get rid of this annotation and make this a private utility.

Reply:
Hmm this would still be needed by users if they're implementing their own Arrow UDFs + using possible-ragged tensor columns. I'll see if I can hide this.